mentors/swift/w.json

```
@@ -1 +1,10 @@
-[]
+[
+{
+  "github_username": "wneumann",
+  "name": "William D. Neumann",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I've been programming for about 40 years, rolling through the ranks of Basic, Pascal, C, Fortran, COBOL, OCaml, and F#. Now I use Swift for most of my programming, having picked it up with the 3.0 release. Also, I can probably eat more tacos than you."
+}
```

```suggestion
[
  {
    "github_username": "wneumann",
    "name": "William D. Neumann",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I've been programming for about 40 years, rolling through the ranks of Basic, Pascal, C, Fortran, COBOL, OCaml, and F#. Now I use Swift for most of my programming, having picked it up with the 3.0 release. Also, I can probably eat more tacos than you."
  }
```
Just some minor indentation changes; feel free to merge this suggestion.